### PR TITLE
docs(website): remove references to @beta

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,15 +14,6 @@ A state controller with its own debugger
 * [Cerebral v1 documentation website](http://cerebral-website.herokuapp.com/)
 * [Cerebral v2 beta documentation website](http://www.cerebraljs.com/)
```
Comment (gaspard) [inline README.md]: Should be Cerebral v2 without 'beta' here...

Comment (henri-hulski) [reply README.md]: Why not remove Cerebral v1.0.2 and call it just Cerebral. Especially as we soon could have Cerebral 3+

Comment (christianalfoni) [reply README.md]: Changed to this:

* [Current Cerebral (2.x and up)](http://www.cerebraljs.com/)
* [Previous Cerebral (1.x)](http://cerebral-website.herokuapp.com/)

**Comment by Guria** [comment]
see also `This project is still in alpha. To test alpha version check` in some READMEs

**Comment by christianalfoni** [comment]
Great catch @Guria ! Think all is good now :)

File: docs/get_started/get_started.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ To install Cerebral BETA you have to explicitly install Cerebral and its depende
```
Comment (henri-hulski) [inline docs/get_started/get_started.md]: Above Cerebral is introduced as beta.

Comment (christianalfoni) [reply docs/get_started/get_started.md]: These docs are not used, should be deleted :)

File: docs/introduction/install.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ To install Cerebral BETA you have to explicitly install Cerebral:
```
Comment (henri-hulski) [inline docs/introduction/install.md]: The line above still contains beta.

Comment (christianalfoni) [reply docs/introduction/install.md]: fixed :)

**Comment by henri-hulski** [comment]
Hmm what's about the beta dependencies of babel preset and pługins?

**Pull request merged**